pkg/virtctl/vmexport/vmexport_test.go

```diff
@@ -89,6 +77,30 @@ var _ = Describe("vmexport", func() {
 			portChan <- uint16(5432)
 			return nil
 		}
+
+		secret = &v1.Secret{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: secretName,
+			},
+			Type: v1.SecretTypeOpaque,
+			Data: map[string][]byte{
+				"token": []byte("test"),
+			},
+		}
+
+		vme = &exportv1.VirtualMachineExport{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: vmeName,
+			},
+			Spec: exportv1.VirtualMachineExportSpec{
+				TokenSecretRef: &secret.Name,
+				Source: v1.TypedLocalObjectReference{
+					APIGroup: &v1.SchemeGroupVersion.Group,
```

Comment:
Can you just use `k8sv1` and remove `v1` import, by chance? I see 2 similar imports:
```
	k8sv1 "k8s.io/api/core/v1"
	v1 "k8s.io/api/core/v1"
```

Reply:
Thanks, good catch. Resolved.